force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -138,16 +177,13 @@ public with sharing class FlowControlLoopsCollections {
      * @return The string "Hello World!; Hello World!; Hello World!; "
      */
     public static String returnHelloWorld() {
-        // Initialize the result String
+       String hw = '';
 
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+         for(Integer i = 1; i <= 3; i++){
+            hw += 'Hello World!; ';
          }
```

Comment:
Good work. For this one I probably would have used 1 = 0; i < 3;.. but this is fine too. 
I also like to use a while loop for this kind of problem. Try to use one if you want the practice.